beluga_ros/cmake/ament.cmake

```diff
@@ -21,6 +22,11 @@ find_package(tf2 REQUIRED)
 find_package(tf2_eigen REQUIRED)
 find_package(tf2_geometry_msgs REQUIRED)
 
+ament_python_install_package(${PROJECT_NAME} PACKAGE_DIR
+                             ${PROJECT_SOURCE_DIR}/tools)
+
+install(DIRECTORY tools USE_SOURCE_PERMISSIONS DESTINATION lib/${PROJECT_NAME})
+
```

Comment:
nit: Move this below, where other installs are found.

Reply:
Done